plugins/DuskVerb/tools/tuner/debug_12k_spike.py

```
@@ -0,0 +1,74 @@
+
+This runs an independent numpy FFT on the raw rendered noiseburst tail and
+reports ABSOLUTE band power (dB) for DV vs anchor at 12902 Hz + neighbours,
+plus the raw-bin shape around 12.9 kHz (sharp narrow peak => Case B).
+"""
+import sys, glob
+import numpy as np
+import soundfile as sf
+
+PAIRS = [
+    ("79 Vocal Chamber",
+     "/tmp/cg_79_Vocal_Chamber/L_noiseburst.wav",
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/cg_79_Vocal_Chamber/L_noiseburst.wav"

(S108)